crates/astria-core/src/sequencerblock/v1alpha1/mod.rs

```diff
@@ -0,0 +1,124 @@
+pub mod block;
+pub mod celestia;
+
+pub use block::{
+    RollupTransactions,
+    SequencerBlock,
+};
+pub use celestia::{
+    CelestiaRollupBlob,
+    CelestiaSequencerBlob,
+};
+use indexmap::IndexMap;
+use sha2::{
+    Digest as _,
+    Sha256,
+};
+
+use crate::{
+    generated::sequencerblock::v1alpha1 as raw,
+    sequencer::v1::{
+        derive_merkle_tree_from_rollup_txs,
+        RollupId,
+    },
+    sequencerblock::Protobuf,
+};
+
+pub(crate) fn are_rollup_ids_included<'a, TRollupIds: 'a>(
```

Comment:
this file is copy pasted from v1

Reply:
Please move this to `lib.rs` or into a utilty module (same for the others) so that we avoid unnecessary code duplication.